pkg/platform/kube/functionres/lazy.go

```diff
@@ -465,10 +466,22 @@ func (lc *lazyClient) deleteRemovedCronTriggersCronJob(functionLabels labels.Set
 		newCronTriggerNames = append(newCronTriggerNames, newCronTriggerName)
 	}
 
+	var cronTriggerInNewCronTriggers string
```

Comment:
would move to another function to reduce noise from the current function main scope